addons/sourcemod/translations/zombieriot.phrases.icons.txt

```diff
@@ -11,37 +11,24 @@
 	"Silvester And Blue Goggles Arrived."
 	{
 		"en" 		"Silvester And Blue Goggles Arrived."
-		"pt"		"Silvester e Blue Goggles chegou."
 		"ru" 		"Сильвестр и Блу Гогглз прибыли."
+		"chi" 		"西尔维斯特与Blue Goggles已来。" 
 	}
 	"Nemesis Arrived."
 	{
 		"en" 		"Nemesis, the original Xeno infector arrived."
-		"pt"		"Nemesis, o Xeno infector original chegou."
 		"ru" 		"Прибыл Немезид, изначальный ксенозаразитель."
+		"chi" 		"复仇女神已来。"
 	}
 	"Arkantos Arrived"
 	{
 		"en" 		"The Atlantean and great leader Arkantos arrived."
-		"pt"		"The Atlantean and great leader Arkantos chegou."
 		"ru" 		"Прибыл Атлант и великий лидер Аркантос."
+		"chi" 		"亚特兰蒂斯之王 ,阿尔坎托斯到达。"
 	}
 	"Sensal Arrived"
 	{
 		"en" 		"The Expidonsan Sensal reaps."
-		"pt"		"O Expidonsan Sensal colhe."
-	}
-	"The Messanger Arrived"
-	{
-		"en" 		"The Messenger has accumulated chaos."
-	}
-	"Chaos Kahmlstein Arrived"
-	{
-		"en" 		"Kahmlstein, the rebellious chaos inflicted leader will annihilate you."
-	}
-	"The Purge Arrived"
-	{
-		"en" 		"The Purge has accumulated chaos."
```

Comment:
How come the translation removals?

Reply:
Ah, my bad. Might have accidentally deleted some lines.

Reply:
Oh wait I see, I was editing a seperate older file saved on my pc which did not have these translations yet; and I just copy pasted the contents assuming it was up-to-date. I'll fix this in a bit.